cli/popper/parser.py

```diff
@@ -155,7 +155,8 @@ def _validate_syntax(self):
             wf_block = list(self._workflow['workflow'].values())[0]
             for key in wf_block.keys():
                 if key not in VALID_WORKFLOW_ATTRS:
-                    log.fail('Invalid attrs found.')
+                    log.fail(
+                        'Invalid workflow attribute {} was found.'.format(key))
```

Comment:
can we single quote the value, i.e. `"Invalid workflow attribute '{}' was found"`?